go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/gin-gonic/gin v1.6.3
+	github.com/joho/godotenv v1.3.0
```

Comment:
We forgot to remove this

Reply:
I thought the compiler would throw an error if something wasn't used. I wonder why that isn't the case here.